notifications_utils/clients/zendesk/zendesk_client.py

```diff
@@ -92,3 +97,95 @@ def send_ticket_data_to_zendesk(self, data):
         ticket_id = response.json()['ticket']['id']
 
         current_app.logger.info(f"Zendesk create ticket {ticket_id} succeeded")
+
+
+class NotifySupportTicket:
+    PRIORITY_URGENT = 'urgent'
+    PRIORITY_HIGH = 'high'
+    PRIORITY_NORMAL = 'normal'
+    PRIORITY_LOW = 'low'
+
+    TYPE_PROBLEM = 'problem'
+    TYPE_INCIDENT = 'incident'
+    TYPE_QUESTION = 'question'
+    TYPE_TASK = 'task'
+
+    TAGS_P2 = 'govuk_notify_support'
+    TAGS_P1 = 'govuk_notify_emergency'
+
+    TYPE_PROBLEM = 'problem'
+    TYPE_INCIDENT = 'incident'
+    TYPE_QUESTION = 'question'
+    TYPE_TASK = 'task'
```

Comment:
This was duplicated deliberately, but was only meant to be temporary . These constants are needed to create the JSON we send to Zendesk. The JSON was being built in `ZendeskClient`, but the "new way" is to do that in `NotifySupportTicket` instead.

The problem at the moment is that tickets will be created using both the new and old way until we switch them all over. For go live tickets (which use the new way), we want the constants here, but everywhere else in the code still expects them to still be in the `ZendeskClient` class. There are three options:

1. Keep the duplication for now 
2. Remove the constants from `ZendeskClient`. This will be a breaking change because we'll have to change the existing ticket code in lots of places in order to keep it working, e.g. [here](https://github.com/alphagov/notifications-api/blob/312a8958222b99d23b58f1cac026bef966c56fd4/app/celery/scheduled_tasks.py#L220)
3. Remove the constants from `NotifySupportTicket` temporarily until all tickets are being created using the new way. No breaking changes, but that does break the separation of the two classes a bit. `NotifySupportTicket` would have to look up get the constants from the `ZendeskClient` in order to get the data to make a ticket.

That's a long winded way of saying I went with 1, but do you think 2 or 3 is better instead? Happy to change if you prefer 🙂